app/models/card.rb

```diff
@@ -1,4 +1,5 @@
 class Card < ActiveRecord::Base
+  validates_uniqueness_of :original_text
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.